Update and alphabetize OpenRouter providers list

cloak1505 opened the pull request:
~~I noticed Cohee removed dead providers in 0ca4cc08bb8f429380fbee4eaa3a483b093e2cb8 which I believe is a good change. However, synchronizing is no longer a matter of copying the list from [the docs](https://openrouter.ai/docs/features/provider-routing#json-schema-for-provider-preferences) and changing double quotes to single quotes. Maybe I'm ignorant and there's a better way to retrieve the list.~~

~~By alphabetizing the list for maintainability, a contributor may simply notice an addition (or removal) of a provider in the sidebar of https://openrouter.ai/models then find the name from the docs linked above and insert it in the nicely ordered list.~~ (Never mind.)

Added: Cerebras, Inception, InoCloud, NextBit
Removed: Reflection

## Checklist:

- [X] I have read the [Contributing guidelines](https://github.com/SillyTavern/SillyTavern/blob/release/CONTRIBUTING.md).

cloak1505 committed:
Update and alphabetize OpenRouter providers list

Cohee1207 reviewed (changes requested):
I disagree. It takes more mental effort to find a difference and decreases maintainability. 
Let's continuing nagging OpenRouter for an API with a list of providers.

cloak1505 commented:
Was the list in OR's doc temporarily pruned but reverted later on? Since I can't remember. Or how were you doing that last commit?

Cohee1207 commented:
Last commit was copy and paste from the docs with lint run afterwards.

cloak1505 committed:
Actual copy and paste

Turns out the doc is already alphabetized but with dead providers moved to the top, so I didn't have to alphabetize the whole list and manually remove the dead ones.

cloak1505 commented:
It just occurred to me that the doc already has it alphabetized at this moment. Dead providers are simply moved to the top, so basically this ends up a copy and paste anyway, starting from AI21 and down.
